CMakeLists.txt

```diff
@@ -1520,6 +1520,18 @@ if(CONFIG_OUTPUT_PRINT_MEMORY_USAGE)
       ${memusage_byproducts}
       )
   endif()
+
+  if(CONFIG_BUILD_OUTPUT_HEX)
+    set(memusage_executable_path "${PROJECT_BINARY_DIR}/${KERNEL_HEX_NAME}")
+  elseif(CONFIG_BUILD_OUTPUT_BIN)
+    set(memusage_executable_path "${PROJECT_BINARY_DIR}/${KERNEL_BIN_NAME}")
+  endif()
```

Comment:
There are several output formats: `BIN EFI EXE HEX S19 UF2`. Multiple can be enabled at once.

Should we print all of them, or can we settle for printing `KERNEL_ELF_NAME` only, which will always exist?

Reply:
Works for me, and then this will be less lines as well.
I think the most important part here is the `PROJECT_BINARY_DIR` either way, as it spells out the directory name, so we know which image is built.
Do others agree with `KERNEL_ELF_NAME` ?